testplan/web_ui/testing/src/AssertionPane/AssertionTypes/GraphAssertions/graphUtils.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Helper functions used to add customisability and styling to the graph and chart assertions.
+ *
+ */
+
+export function returnStyle(graph_type){
+    if(graph_type === 'Contour'){
+       return(
+                {
+                    stroke: '#125C77',
+                    strokeLinejoin: 'round'
+                }
+        );
+    }
+}
+
+export function returnRadius(graph_type){
+    if(graph_type === 'Hexbin'){
+       return(10);
+    }
+}
+
+export function returnXType(graph_type){
+    if(graph_type ==='Bar'){
+       return("ordinal");
+    }
+}
+
+export function returnColour(options){
```

Comment:
So what happens if `options` is specified but has no `colour` attribute?

Reply:
The function does not throw an error, but does not return anything, so react-vis assigns it's default colours, i will fix this to also return a random colour. 
(I also want to improve the colours so instead of random colour, its a more sensible set of defaults like react-vis, e.g dark blues and greens etc)